lib/routes/xiaoyuzhou/podcast.ts

```diff
@@ -67,17 +68,29 @@ async function handler(ctx) {
         }
     }
 
-    const episodes = page_data.props.pageProps.podcast.episodes.map((item) => ({
+    let episodes = page_data.props.pageProps.podcast.episodes.map((item) => ({
         title: item.title,
         enclosure_url: item.enclosure.url,
         itunes_duration: item.duration,
         enclosure_type: 'audio/mpeg',
         link: `https://www.xiaoyuzhoufm.com/episode/${item.eid}`,
         pubDate: parseDate(item.pubDate),
-        description: item.description.replaceAll('\n', '<br>') || item.shownotes,
         itunes_item_image: (item.image || item.podcast?.image)?.smallPicUrl,
     }));
 
+    episodes = await Promise.all(
+        episodes.map((item) =>
+            cache.tryGet(item.link, async () => {
+                const eid = item.link.match(/episode\/([a-zA-Z0-9]+)/)[1];
```

Comment:
`eid` is provided in episodes https://github.com/DIYgod/RSSHub/blob/c3971672831925a2a924b07ab5447ffb4ec00513/lib/routes/xiaoyuzhou/podcast.ts#L76
You can add an extra property `eid` in https://github.com/DIYgod/RSSHub/blob/c3971672831925a2a924b07ab5447ffb4ec00513/lib/routes/xiaoyuzhou/podcast.ts#L71-L79 so that you can use it directly without string matching